Automattic/wp-calypso
# My Home: New site setup list for experimental layout

#### Changes proposed in this Pull Request

<img width="1423" alt="Screen Shot 2020-04-29 at 2 29 35 PM" src="https://user-images.githubusercontent.com/1270189/80648989-0675df80-8a26-11ea-8ab4-170658e12180.png">

Adds a Site Setup List component that visually works well with the new My Home Layout.

#### Testing instructions
- Checkout this branch
- Create a new site
- Visit `http://calypso.localhost:3000/home/:siteId:?flags=home/experimental-layout`
- Verify that Site Setup Steps look visually good, no regressions in task behavior, and that analytics events still fire
